doc/getting-started/install.md

```diff
@@ -139,13 +139,7 @@ Checkout the latest version of cardano-node (choose the tag with the highest ver
 
 We explicitly use the GHC version that we installed earlier.  This avoids defaulting to a system version of GHC that might be older than the one you have installed.
 
-    cabal configure --with-compiler=ghc-8.10.2
-
-Update the local project file to use the VRF library that you installed earlier.
-
-    echo "package cardano-crypto-praos" >>  cabal.project.local
-    echo "  flags: -external-libsodium-vrf" >>  cabal.project.local
-
```

Comment:
It's better if we preserve this part, but add a comment that this is only for a development build.

btw, I don't know yet how to make it work without it.

Reply:
Thanks, a note is now added and that part is back in.